deploy/olm-catalog/keepalived-operator/0.2.2/keepalived-operator.v0.2.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,298 @@
+          - watch
+          - create
+          - patch
+        serviceAccountName: keepalived-operator
+      deployments:
+      - name: keepalived-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: keepalived-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: keepalived-operator
+            spec:
+              containers:
+              - command:
+                - keepalived-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: KEEPALIVED_OPERATOR_IMAGE_NAME
+                  value: quay.io/redhat-cop/keepalived-operator:v0.2.2
+                - name: KEEPALIVEDGROUP_TEMPLATE_FILE_NAME
+                  value: /templates/keepalived-template.yaml
+                - name: OPERATOR_NAME
+                  value: keepalived-operator
+                image: quay.io/redhat-cop/keepalived-operator:v0.2.2
+                imagePullPolicy: Always
+                name: keepalived-operator
+                resources: {}
+              serviceAccountName: keepalived-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          - pods
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - services/finalizers
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          - deployments
+          verbs:
+          - get
+          - list
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resourceNames:
+          - keepalived-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        serviceAccountName: keepalived-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - loadbalancer
+  - keepalived
+  - ipfailover
+  - externalIP
+  - self-hosted
+  links:
+  - name: repository
+    url: https://github.com/redhat-cop/keepalived-operator
+  - name: containerImage
+    url: https://quay.io/redhat-cop/keepalived-operator:v0.2.2
+  maintainers:
+  - email: rspazzol@redhat.com
+    name: Raffaele Spazzoli
+  maturity: alpha
+  provider:
+    name: Containers & PaaS CoP
```

Comment:
Please update to "Red Hat Community of Practice"

Reply:
done